src/transform/frontmatter/emplace.ts

```diff
@@ -0,0 +1,26 @@
+import {dump} from 'js-yaml';
+
+import {FrontMatter, frontMatterFence, unescapeLiquidSubstitutionSyntax} from './common';
+
+export const serializeFrontMatter = (frontMatter: FrontMatter) => {
```

Comment:
is the problem we are solving in this PR valid only for meta or for all yaml pages too?